kolibri/content/views.py

```diff
@@ -58,3 +58,31 @@ def get(self, request, zipped_filename, embedded_filepath):
         response["Accept-Ranges"] = "none"
 
         return response
+
+class DownloadContentView(View):
+
+    def get(self, request, filename, new_filename):
+        """
+        Handles GET requests and serves a static file as an attachment.
+        """
+
+        # calculate the local file path of the file
+        path = get_content_storage_file_path(filename)
+
+        # if the file does not exist on disk, return a 404
+        if not os.path.exists(path):
+            raise Http404('"%(filename)s" does not exist locally' % {'filename': filename})
+
+        # generate a file response
+        response = FileResponse(open(path, 'rb'))
+
+        # set the content-type
+        response['Content-Type'] = mimetypes.guess_type(new_filename)[0]
+
+        # set the content-disposition as attachment to force download
+        response['Content-Disposition'] = 'attachment; filename=%s' % new_filename
```

Comment:
(In that case, the URL pattern wouldn't change, but we'd be ignoring the `new_filename` piece on the server side.